### PR TITLE
another brew commands

### PR DESCRIPTION
**Pull request by fate-7**
My usual brew command

## Type of change

- [x] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by fate-7** [comment]
add brew commands

File: commands/developer-utils/brew/brew-update.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# @raycast.mode fullOutput
+# @raycast.packageName Brew
+#
+# Optional parameters:
+# @raycast.icon 🍺
+#
+# Documentation:
+# @raycast.description Run Brew Update
+# @raycast.author chengzhiqi
+# @raycast.authorURL https://twitter.com/tholanda
```
Comment (unnamedd) [inline commands/developer-utils/brew/brew-update.sh:15]: @fate-7, please, change the value to your social media

File: commands/developer-utils/brew/brew-list.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# @raycast.mode fullOutput
+# @raycast.packageName Brew
+#
+# Optional parameters:
+# @raycast.icon 🍺
+#
+# Documentation:
+# @raycast.description Show Brew List
+# @raycast.author chengzhiqi
+# @raycast.authorURL https://twitter.com/tholanda
```
Comment (unnamedd) [inline commands/developer-utils/brew/brew-list.sh:15]: @fate-7, please, change the value to your social media

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hi @fate-7, thanks for your contribution.

I’ve made just two comments and basically I’m asking to change the `authorURL` to your social media, because that twitter there is mine! :D

**Review by fate-7 — COMMENTED** [review]
Change to my personal address

**Review by unnamedd — APPROVED** [review]
Looks good to me! 
It's approved! ✅

**Pull request merged**